.circleci/config.yml

```diff
@@ -108,5 +109,9 @@ workflows:
             branches:
               only: master
     jobs:
-      - unit_test
-      - integration_test
+      - unit_test:
+          context:
+            - Gruntwork Admin
+      - integration_test:
+          context:
+            - Gruntwork Admin
```

Comment:
Hm, can nightly jobs use contexts? I don't think they can. See https://gruntwork-io.slack.com/archives/CLYE0JYS0/p1606143137141900?thread_ts=1602848098.207000&cid=CLYE0JYS0 for discussion.

Reply:
Thanks, caught up on that discussion + docs, and repaired this PR to exclude the nightly jobs from the context addition.